tiledb/sm/query/readers/sparse_unordered_with_dups_reader.cc

```diff
@@ -253,8 +253,7 @@ void SparseUnorderedWithDupsReader<BitmapType>::load_tile_offsets_data() {
       tile_offsets_min_frag_idx_ == std::numeric_limits<unsigned>::max() &&
       tile_offsets_max_frag_idx_ == 0;
   uint64_t available_memory =
-      array_memory_tracker_->get_memory_available() -
-      array_memory_tracker_->get_memory_usage(MemoryType::TILE_OFFSETS);
+      array_memory_tracker_->get_memory_available();
```

Comment:
get_memory_available should be changed so that tile offsets from the new counters are subtracted.

Reply:
I have done that. Adding a comment now to show where